gulpfile.js

```diff
@@ -0,0 +1,249 @@
+var gulp = require("gulp"),
```

Comment:
I had started working on some changes to the `gulpfile` in the `slamdata` project - I had some issues with getting it to build, maybe just a Windows thing as I've not tried it on a Mac yet... but aside from that, I switched to `psc-make` with `browserify` too, as it can make for much faster incremental compiles. I can break that out into its own project perhaps.